### PR TITLE
Disable creation

### PR DESCRIPTION
**Pull request by mikiquantum**
Commented multi-account creation extrinsic code.
Ensuring tests still pass by keeping the create function as internal only, for smoother transition.

### CODE REVIEW

**Review by philipstanislaus — APPROVED** [review]
Thanks 🙏

**Comment by vedhavyas** [comment]
@mikiquantum would removing this extrinsic changes/breaks the metadata and makes the signer unusable? If so, I was thinking we could keep the extrinsic as is but just error out with a helpful error message and do a metadata update once we add the proxy pallet in. WDYT ?

**Comment by mikiquantum** [comment]
@vedhavyas I like that approach too, I would then create an empty extrinsic function that returns a useful error. Since I am not sure if that would affect the parity signer, although in any case, that would need to be updated with the bundle of RC3 I believe.

File: src/lib.rs [hunk 1]
```diff
@@ -301,38 +304,12 @@ decl_module! {
...
-                threshold,
-                signatories,
-                deposit,
-                depositor: who.clone(),
-            });
-
-            Self::deposit_event(RawEvent::NewMultiAccount(who, id));
+            ensure!(false, Error::<T>::DeprecatedPallet);
 
             Ok(())
```
Comment (vedhavyas) [inline src/lib.rs]:
```suggestion
            Err(Error::<T>:: DeprecatedPallet.into())
```

With this, we can remove the ensure check above

**Review by vedhavyas — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -2,10 +2,6 @@
 # will have compiled files and executables
 /target/
 
-# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
-# More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
-Cargo.lock
```
Comment (mikiquantum) [inline .gitignore:7]: https://github.com/paritytech/substrate/pull/6676

**Pull request merged**